### PR TITLE
Issue#53 show markers locations and results

### PR DESCRIPTION
**Pull request by LChoiSH**

### CODE REVIEW

File: src/frontend/src/App.js [hunk 1]
```diff
@@ -16,226 +16,148 @@ import SearchIcon from '@material-ui/icons/Search';
...
   };
 
-  makeuser = () => {
-    let users = this.state.users;
-    users.push({
+  setMarker = (direction) => {
+    if(this.state.selectedMarker == -1) return;
+
+    let newMarkers = clone(this.state.markers);
+
```
Comment (Jelldo) [inline src/frontend/src/App.js]: Redundant spaces

File: src/frontend/src/App.js [hunk 2]
```diff
@@ -16,226 +16,148 @@ import SearchIcon from '@material-ui/icons/Search';
...
 
     return ;
   };
 
-  makeuser = () => {
-    let users = this.state.users;
-    users.push({
+  setMarker = (direction) => {
+    if(this.state.selectedMarker == -1) return;
+
```
Comment (Jelldo) [inline src/frontend/src/App.js]: redundant space

File: src/frontend/src/FindBox.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+  handleChange = (e) => {
+    this.props.changeLoc({
+      name: e.target.name,
+      value: e.target.value,
+      index: e.target.id
+    });
+  }
+
+  makeInput() {
```
Comment (Jelldo) [inline src/frontend/src/FindBox.js]: I think _setCoordinates_ looks more recognizable and intuitive.

File: src/frontend/src/FindBox.js [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+          </Paper>
+          <br></br>
+        </div>
+      )
+    }
+
+    return output;
+  };
+
+  showResult() {
```
Comment (Jelldo) [inline src/frontend/src/FindBox.js]: showAreaResult or showLocationResult

need to name it clearer that makes we know what type of data this function returns.

File: src/frontend/src/FindBox.js [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+          </div>
+
+        </div>
+      )
+    };
+
+    return output;
+  };
+
+  showUsersResult(areaNum) {
```
Comment (Jelldo) [inline src/frontend/src/FindBox.js]: showUserTravelInfo or showUserTravelResult

We use 'userTravelInfo' in backend, so using this variable name would be helpful

File: src/frontend/src/Map.js [hunk 6]
```diff
@@ -5,36 +5,39 @@ class Map extends React.PureComponent {
...
+  onMarkerChange = (event) => {
+    this.props.setMarker({
+      lat: event.latLng.lat(),
+      lng: event.latLng.lng()
     });
+
   }
 
   showMarkers() {
     let output = [];
```
Comment (Jelldo) [inline src/frontend/src/Map.js]: let _markers_

File: src/frontend/src/FindBox.js [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+    let output = [];
```
Comment (Jelldo) [inline src/frontend/src/FindBox.js]: let _userTravelInfos_ or anything clear

File: src/frontend/src/FindBox.js [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+      for (let j =0; j<this.props.areas[areaNum].users[i].route.length; j++) {
+        output.push(
+          <div>
+            {(j+1) + ". " + (this.props.areas[areaNum].users[i].route[j].transportation == "driving" ?
+              this.props.areas[areaNum].users[i].route[j].name
+              : " ( " + this.props.areas[areaNum].users[i].route[j].transportation + " " + this.props.areas[areaNum].users[i].route[j].lineNum + " ) " + this.props.areas[areaNum].users[i].route[j].startName + " -> " + this.props.areas[areaNum].users[i].route[j].endName)}
+          </div>
+        )
+      }
+
```
Comment (Jelldo) [inline src/frontend/src/FindBox.js]: extra space

File: src/frontend/src/App.js (hunk 1 above)
Comment (Jelldo) [inline src/frontend/src/App.js]: extra space

File: src/frontend/src/FindBox.js [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+  handleChange = (e) => {
+    this.props.changeLoc({
+      name: e.target.name,
+      value: e.target.value,
+      index: e.target.id
+    });
+  }
+
+  makeInput() {
+    let output = [];
```
Comment (Jelldo) [inline src/frontend/src/FindBox.js]: let _coordinates_

File: src/frontend/src/FindBox.js [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+        </div>
+      )
+    }
+
+    return output;
+  };
+
+  showResult() {
+    let result = this.props.areas;
+    let output = [];
```
Comment (Jelldo) [inline src/frontend/src/FindBox.js]: let _groupTravelInfo_ or anything clear

File: src/frontend/src/Map.js (hunk 6 above)
Comment (Jelldo) [reply src/frontend/src/Map.js]: and remove spaces here and there 🗡

File: src/frontend/src/App.js [hunk 11]
```diff
@@ -16,226 +16,148 @@ import SearchIcon from '@material-ui/icons/Search';
...
-
-    for (var i=0; i<this.state.areas[areaNum].users.length; i++) {
-      output.push(
-        <div>
-          <h2>{(i+1) + "번째 유저"}</h2>
-          <div>{"소요시간" + this.state.areas[areaNum].users[i].duration}</div>
-          <div>{"거리" + this.state.areas[areaNum].users[i].distance}</div>
-        </div>
-      )
+  changeLoc = (direction) => {
```
Comment (Jelldo) [inline src/frontend/src/App.js]: changeLocation
we don't abbreviate 🔪

File: src/frontend/src/App.js (hunk 1 above)
Comment (LChoiSH) [reply src/frontend/src/App.js]: Because it is declaration

File: src/frontend/src/FindBox.js (hunk 8 above)
Comment (LChoiSH) [reply src/frontend/src/FindBox.js]: for for statements

**Pull request merged**